srcpkgs/diffuse/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'diffuse'
+pkgname=diffuse
+version=0.4.8
+revision=1
+archs=noarch
+build_style=fetch
+depends="pygtk python"
```

Comment:
We are no longer accepting python2 packages. Is there any chance of getting it to work with python3?

Reply:
It seems like the author has stopped working on this project few years ago. I'm also not familiar with projects code, so can't say anything about porting to py3.

What do you think of this situation?

Reply:
python2 and abandoned projects are both blockers for inclusion

Reply:
Okay. Completely understand.

I've found fork of this tool (with py3 & gtk3) - made a template for latest release, but it has annoying bug (fixed in later commits).

I asked for spawning new release (with bugfix) [here](https://github.com/MightyCreak/diffuse/issues/42) - it's still open.

I can't say how long would it take to bring that fork into the void-packages, so what is the best way here? - Close this PR and create new one later? Or leave this open until fresh release will be spawned in that repo (py3-fork)?

Reply:
If the fix is tiny (one can hope), you can create a patch and package their latest release. Try to ask them about it, perhaps. That way they don't have to cut a release.

Reply:
Okay. I'll play with the "patch" approach on holidays and come back with some suitable solution. Thanks.

Reply:
Updated PR with Py3-version of this application (also changed the title). Used fresh release (with bug-fixes). It works on my machine.

I hope now it's package the proper way. What would you say? - Can we merge it?